docs/specs/clients/notify/rpc-methods.md

```diff
@@ -39,8 +39,7 @@ sequenceDiagram
 ```
 
 Notes:
-- Each client will call `wc_notifyWatchSubscriptions` separately with separate persistent keys kY
-- Recommended: persist kY to avoid multiple unnecessary response topics
+- Each client will call `wc_notifyWatchSubscriptions` to receive subscription updates for a blockchain account
```

Comment:
```suggestion
- Generate and persist kY per req topic and blockchain account
- Each client will call `wc_notifyWatchSubscriptions` to receive subscription updates for a blockchain account
```

Reply:
See latest version. I actually removed this detail from these bullet points in-favor of below in the method description.